userpatches/customize-image.sh

```diff
@@ -246,6 +232,42 @@ uri=http://detectportal.firefox.com/canonical.html
 interval=300
 NMCONF
 
+# ============================================================================
+# Modify config.txt to put GPIOs into correct state for OpenWB
+# ============================================================================
+apt-get install -y --no-install-recommends labwc wayfire seatd xdg-user-dirs firefox-esr swayidle wlopm
+
+mkdir -p /home/admin/.config/labwc || true
+cat >/home/admin/.config/labwc/autostart <<'LABWCAUTOSTART'
+/usr/bin/firefox --kiosk http://localhost:7070/ &
+/usr/bin/swayidle -w timeout 600 'wlopm --off \*' resume 'wlopm --on \*' &
+LABWCAUTOSTART
+
+mkdir -p /home/admin/.config/systemd/user || true
+cat >/home/admin/.config/systemd/user/kiosk.service <<'KIOSKSERVICE'
+[Unit]
+Description=Start Kiosk mode
+[Service]
+Type=simple
+ExecStart=/usr/bin/labwc
+[Install]
+WantedBy=default.target
+KIOSKSERVICE
+
+# Enable auto-start of kiosk mode
+mkdir -p /home/admin/.config/systemd/user/default.target.wants || true
+ln -sf /home/admin/.config/systemd/user/kiosk.service /home/admin/.config/systemd/user/default.target.wants/kiosk.service || true
+
+cp -r /etc/skel/. /home/admin/ 2>/dev/null 
+chown -R admin:admin /home/admin
+chmod 755 /home/admin
+
+# Fake loginctl enable-linger admin
+mkdir -p /var/lib/systemd/linger/ || true
+touch /var/lib/systemd/linger/admin
+
+# Disable autologin
+rm /etc/systemd/system/getty@.service.d/override.conf
```

Comment:
Currently armbian seems to auto-login root on the console after boot, which I see as a security risk. So I'm disabling this.